src/redux/user/user.actions.js

```diff
@@ -35,6 +36,11 @@ const loginUser = (payload) => ({
   payload
 });
 
+const loginByGoogle = (payload) => ({
+  type: GOOGLE_USER,
```

Comment:
You need to use a more understandable type name for this action (like **LOGIN_USER_BY_GOOGLE** or **LOGIN_BY_GOOGLE**)

Reply:
Fixed